app/src/main/res/layout/activity_hospital.xml

```diff
@@ -218,6 +218,13 @@
 
     </LinearLayout>
 
+    <TextView
+        android:id="@+id/amountofoverduedevices"
```

Comment:
please use camelCase or snake_case for naming

Reply:
done.